src/main/kotlin/org/openmbee/flexo/mms/AccessControl.kt

```diff
@@ -102,34 +108,25 @@ fun permittedActionSparqlBgp(permission: Permission, scope: Scope, find: Regex?=
             ?policy a mms:Policy ;
                 mms:scope ?scope ;
                 mms:role ?role ;
+                mms:subject ?__mms_policyAgent ;
                 .
         }
 
-        {   
-            # the policy applies to this user within an appropriate scope
-            graph m-graph:AccessControl.Policies {
-                # policy about user
-                ?policy mms:subject mu: .
-            }
-    
+        {
+            # set user as the agent
+            bind(mu: as ?__mms_policyAgent)
```

Comment:
the changes here are orthogonal to Artifacts. it just simplifies the auth SPARQL queries